packages/smooth_app/lib/pages/product/add_basic_details_page.dart

```diff
@@ -0,0 +1,185 @@
+        key: _formKey,
+        child: ListView(
+          children: <Widget>[
+            Align(
+              alignment: Alignment.topLeft,
+              child: ProductImageCarousel(
+                widget.product,
+                height: size.height * 0.20,
+                onUpload: (_) {},
+              ),
+            ),
+            SizedBox(height: size.height * 0.02),
+            if (widget.product.barcode != null)
+              Padding(
+                padding: EdgeInsets.symmetric(horizontal: size.width * 0.05),
+                child: Column(
+                  children: <Widget>[
+                    Text(
+                      '${appLocalizations.barcode}: ${widget.product.barcode!}',
+                      style: TextStyle(
+                        color: colorScheme.onBackground,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    SizedBox(height: size.height * 0.02),
+                    SmoothTextFormField(
+                      controller: _productNameController,
+                      type: TextFieldTypes.PLAIN_TEXT,
+                      hintText: appLocalizations.product_name,
+                      validator: (String? value) {
+                        if (value == null || value.isEmpty) {
+                          return appLocalizations
+                              .add_basic_details_product_name_error;
+                        }
+                        return null;
+                      },
+                    ),
+                    SizedBox(height: size.height * 0.02),
+                    SmoothTextFormField(
+                      controller: _brandNameController,
+                      type: TextFieldTypes.PLAIN_TEXT,
+                      hintText: appLocalizations.brand_name,
+                      validator: (String? value) {
+                        if (value == null || value.isEmpty) {
+                          return appLocalizations
+                              .add_basic_details_brand_name_error;
+                        }
+                        return null;
+                      },
+                    ),
+                    SizedBox(height: size.height * 0.02),
+                    SmoothTextFormField(
+                      controller: _weightController,
+                      type: TextFieldTypes.PLAIN_TEXT,
+                      hintText: appLocalizations.weight_gram,
+                      textInputType: TextInputType.number,
+                      validator: (String? value) {
+                        if (value == null || value.isEmpty) {
+                          return appLocalizations
+                              .add_basic_details_weight_gram_error;
+                        }
+                        return null;
+                      },
+                    ),
+                    SizedBox(height: size.height * 0.05),
+                  ],
+                ),
+              ),
+            Row(
```

Comment:
I don't see how you put the buttons cleanly at the bottom of the page. Here they are just after the rest, aren't they?

Reply:
I used Row to horizontally align the buttons and used `MainAxisAlignment.spaceAround` value in `mainAxisAlignment` for spacing purposes. Please suggest if you have something else in your mind.

Reply:
My concern is about the vertical position of the Row. Let's keep it that way.